Not stable iOS build for features with BuildCocoapod-task

There is often error on task cocoapodBuild___ for project included pods both in mpp-library and feature. 

Like this:
```
FAILURE: Build failed with an exception.
117 * What went wrong:
118 Execution failed for task ':mpp-library:cocoapodBuildBindAccountIosIphoneosArm64'.
119 > A problem occurred starting process 'command 'xcodebuild''
```

Build can complete success after some retry attempt. Looks like the reason in :feature:cocoapodBuild and :mpp-library:cocoapodBuild starts in the same time.

Tried to fix with adding deps in mpp-library/build.grable.kts:


```
val bindAccountProject = subprojects.first { it.name == "bindAccount" }

tasks
    .matching { it.name.startsWith("cocoapodBuild") }
    .whenTaskAdded {
        val mppTask = this
        bindAccountProject.tasks
            .matching { it.name == name }
            .whenTaskAdded {
                println("$mppTask dependsOn $this")
                mppTask.dependsOn(this)
            }
    }
```

But no result

--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ repositories {
 }
 
 dependencies {
-    implementation("dev.icerock:mobile-multiplatform:0.5.0")
+    implementation("dev.icerock:mobile-multiplatform:0.5.1")
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.61")
     implementation("com.android.tools.build:gradle:3.5.3")
 }

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -36,7 +36,7 @@ publishing {
         register("plugin", MavenPublication::class) {
             groupId = "dev.icerock"
             artifactId = "mobile-multiplatform"
-            version = "0.5.0"
+            version = "0.5.1"
 
             from(components["java"])
         }

--- a/src/main/kotlin/dev/icerock/gradle/MobileMultiPlatformPlugin.kt
+++ b/src/main/kotlin/dev/icerock/gradle/MobileMultiPlatformPlugin.kt
@@ -136,7 +136,7 @@ linkerOpts = -framework ${pod.module}
             KonanTarget.IOS_X64 -> "iphonesimulator" to "x86_64"
             else -> throw IllegalArgumentException("${kotlinNativeTarget.konanTarget} is unsupported")
         }
-        val cocoapodsOutputDir = File(project.rootProject.buildDir, "cocoapods")
+        val cocoapodsOutputDir = File(project.buildDir, "cocoapods")
         val capitalizedPodName = pod.capitalizedModule
         val capitalizedSdk = arch.first.capitalize()
         val capitalizedArch = arch.second.capitalize()

